test/factory/common_factory_tests.dart

```diff
@@ -103,17 +112,43 @@ void commonFactoryTests(ReactComponentFactoryProxy factory, {bool isFunctionComp
         shouldAlwaysBeList: isDartComponent2(factory({})));
   });
 
-  // Skipped for Function Components because they dont have instance members ... they are functions.
```

Comment:
Needed to get creative here to work around this restriction; ended up using the `onDartRender` callback to capture the actual children passed to render via props